pkg/runtime/pubsub/subscriptions.go

```diff
@@ -41,11 +41,12 @@ const (
 
 type (
 	SubscriptionJSON struct {
-		PubsubName string            `json:"pubsubname"`
-		Topic      string            `json:"topic"`
-		Metadata   map[string]string `json:"metadata,omitempty"`
-		Route      string            `json:"route"`  // Single route from v1alpha1
-		Routes     RoutesJSON        `json:"routes"` // Multiple routes from v2alpha1
+		PubsubName      string            `json:"pubsubname"`
+		Topic           string            `json:"topic"`
+		DeadLetterTopic string            `json:"dead_letter_topic"`
```

Comment:
Should be `deadLetterTopic` to be consistent with the subscription CRD definition.

Reply:
Done